### ISSUE TITLE
PHP7.2

### ISSUE THREAD

**Issue by BenasPaulikas** [yiisoft/yii2-mongodb#223]
Hi looks like this: https://github.com/yiisoft/yii2/pull/14478 is not done here ?

### What steps will reproduce the problem?

Install php7.2

### What's expected?

No error

### What do you get instead?
```
{
  "name": "PHP Compile Error",
  "message": "Cannot use yii\\base\\Object as Object because 'Object' is a special class name",
  "code": 64,
  "type": "yii\\base\\ErrorException",
  "file": "/var/www/html/alua-yii/vendor/yiisoft/yii2-mongodb/Database.php",
  "line": 10,
  "stack-trace": [
    "#0 [internal function]: yii\\base\\ErrorHandler->handleFatalError()",
    "#1 {main}"
  ]
}
```
### Additional info

| Q                      | A
| ---------------------- | ---
| Yii version            | dev-master
| Yii MongoDB version    |dev-master
| MongoDB server version |1.2.9
| PHP version            |7.2
| Operating system       |Linux mint

**Comment by klimov-paul** [comment]
There is nothing can be done withing an extension untill Yii 2.0.13 release.

**Comment by klimov-paul** [comment]
Resolved by commit d91a09a6aacc698fd3509dfad7f95d3460a85b7e
